app/src/main/java/com/simply/birthdayapp/main/addEvent/presentation/AddEventViewModel.kt

```diff
@@ -21,13 +31,10 @@ class AddEventViewModel : ViewModel() {
     private val _familyRelation = MutableStateFlow(FamilyRelation.getDisplayNames())
     val familyRelation: StateFlow<List<String>> = _familyRelation.asStateFlow()
 
-    private val _imageUrl = MutableStateFlow<Uri?>(null)
-    val imageUrl: StateFlow<Uri?> = _imageUrl.asStateFlow()
-
     private val _selectedDay = MutableStateFlow(Calendar.getInstance().get(Calendar.DAY_OF_MONTH))
     val selectedDay: StateFlow<Int> = _selectedDay.asStateFlow()
 
-    private val _selectedMonth = MutableStateFlow(Calendar.getInstance().get(Calendar.MONTH))
+    private val _selectedMonth = MutableStateFlow(Calendar.getInstance().get(Calendar.MONTH) + 1)
```

Comment:
Adding +1 for the month is necessary because in Calendar months are indexed from 0 (where January is 0, February is 1, and so on until December is 11). If you want to display or store the month in a 1-12 format, you need to add +1 to the month index.